data/json/mutations.json

```diff
@@ -1851,16 +1849,6 @@
         "category" : ["MUTCAT_MEDICAL"]
     },{
         "type" : "mutation",
-        "id" : "M_IMMUNE",
```

Comment:
As far as I know, mutations can't be removed just like that or the game will complain if a game with a character with this mutation is loaded.

It should probably be dummied out instead, at least until the new stable version.